docs/app_model/instancing_apps.md

```diff
@@ -0,0 +1,70 @@
+---
+id: app_model_instancing
+title: Instance applications
+---
+Instancing applications can be done in several ways. Here is the supported ones by NetDaemon.
+
+### Instancing an application using attribute
+Instance any class with the simple attribute like below.
+```csharp
+[NetDaemonApp]
+public class MyApp
+{
+
+}
+```
+If you need to some initialization to be called after all apps are instanced use the `IInitializable` interface.
```

Comment:
```suggestion
If you need to some initialization to be called after the properties are initiaized from the yaml configuration use the `IInitializable` interface.

```

Reply:
The contrustors are run in order of the dependencies so that should not be a problem